torch/distributed/distributed_c10d.py

```diff
@@ -1464,6 +1482,8 @@ def destroy_process_group(group: Optional[ProcessGroup] = None):
 
 def get_rank(group: Optional[ProcessGroup] = None) -> int:
     """
+    Return the rank of the current process.
```

Comment:
This is not necessarily correct since a process can have multiple groups, hence multiple ranks. Can we just keep the description below?

Reply:
Same as my detailed reply above. I can change this to something like `Return the rank of the current process as described below.`here, and similar changes at other places if you think that makes more sense? Thanks!